.github/workflows/lint.yml

```diff
@@ -41,6 +41,26 @@ jobs:
           # use this.
           warnings: "-native-cc"
 
+  yapf:
+    runs-on: ubuntu-18.04
+    env:
+      BASE_REF: ${{ github.base_ref }}
+    steps:
+      - name: Checking out repository
+        uses: actions/checkout@v2
+      - name: Setting up python
+        uses: actions/setup-python@v2
+        # We have to explicitly fetch the base branch as well
+      - name: Fetching Base Branch
+        run: git fetch --no-tags --prune --depth=1 origin "${BASE_REF?}:${BASE_REF?}"
+      - name: Running yapf
+        run: |
+          python3 -m pip install yapf
```

Comment:
Maybe put this back in a separate step now that path stuff is sorted out?

Reply:
Done!